src/cljs/chronograph_web/http.cljs

```diff
@@ -1,6 +1,15 @@
 (ns chronograph-web.http
   (:require [ajax.core :as ajax]))
 
+(defn GET
+  [uri m]
+  (merge {:method :get
+          :uri uri
+          :timeout 8000 ;; TODO: put in config?
```

Comment:
Make it a config.

Reply:
We aren't using aero or mount in cljs yet. Should we take it up as a separate chore?

Reply:
We can just make it a var or a `goog-define` in a `config` namespace for now I think. No need to introduce a dependency.

Reply:
Done. Used goog-define to set the value.